account/test/usecase/createUserAccount.test.js

```diff
@@ -0,0 +1,33 @@
+import app from '../../src/app.js';
+import request from 'supertest';
+import { client, getUsersCollection } from '../../src/repositories/accountRepository.js';
+
+describe('Account Creation', () => {
+  afterEach(async () => {
+    await client.connect();
```

Comment:
are you creating another connection?

Reply:
yes, I did according to the mentor's explanation.